Assets/MixedRealityToolkit.ThirdParty/MRTK-Quest/Scripts/Input/Controllers/OculusQuestHand.cs

```diff
@@ -337,10 +341,30 @@ protected bool UpdateHandData(OVRHand ovrHand, OVRSkeleton ovrSkeleton)
                 }
             }
 
+            // Pinch was also used as grab, we want to allow hand-curl grab not just pinch.
+            // Determine pinch and grab separately
+            MixedRealityPose wristPose, indexKnucklePose, indexTipPose;
+            if (jointPoses.TryGetValue(TrackedHandJoint.Wrist, out wristPose))
+            {
+                if (jointPoses.TryGetValue(TrackedHandJoint.IndexTip, out indexTipPose))
+                {
+                    if (jointPoses.TryGetValue(TrackedHandJoint.IndexKnuckle, out indexKnucklePose))
+                    {
+                        // compare wrist-knuckle to wrist-tip
+                        float wristToIndexTip = Vector3.Distance(wristPose.Position, indexTipPose.Position);
```

Comment:
Instead of distance checks, which perform a square root operation, I recommend using sqrMagnitude, and squarring your distance threshold value.

Reply:
(performance nit)

Reply:
yup we talked about this actually - but I was a little lazy here sorry

Reply:
Id like to see the sqrMagnitudes done before I merge this in. Appreciate you taking the time to write this up :)